### ISSUE TITLE
Focus issue ?

### ISSUE THREAD

**Issue by itwars** [chinchang/cta.js#3]
Hello Gour,

First thanks a lot for your awesome code, anyway I face to a strange issue, while using key event instead of mouse : 

- If you F5 to refresh your browser, and then press "esc" to fire the event I don't have the transition
- if you F5, then click inside the browser window, then "esc" I got the expecting transition 

Strange ....

```
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>cta.js - Animate your 'action-to-effect' paths!</title>
  <meta name="author" content="Kushagra Gour">
  <meta name="description" content="Animate any element onto another element.">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <style>
   .e1 {
      position: absolute;
      background: #F90;
      border-radius: 50%;
      margin-left: -10px;
      margin-top: -10px;
      left: 10px;
      top: 430px;
      width: 20px;
      height: 20px;
    }
   .e2 {
      position: fixed;
      top: 0;
      bottom: 0;
      left: 0;
      right: 0;
      padding: 40px;
      background: #EA006F;
      color: white;
      z-index: 1;
      transition: 200ms ease;
    }
   .hide {
      visibility: hidden;
      opacity: 0;
    }
  </style>
</head>

<body>
  <div class="e1"></div>
  <div class="e2 hide"></div>
  <script src="src/cta.js"></script>
  <script>
   addEventListener('keyup', function(e) {
     if (e.which === 27) {
      var e1 = document.querySelector('.e1'),
       e2 = document.querySelector('.e2');
      e1.classList.add('hide');
      cta(e1, e2, {
         duration: 3
       },
       function() {
         e2.classList.remove('hide');
       });
     }
   }, false);
  </script>
</body>

</html>
```

Issue on JSBin: http://jsbin.com/sebalilana

**Comment by chinchang** [comment]
@itwars Interesting. Looking into it...

**Comment by chinchang** [comment]
@itwars Yes, this seems like an issue in Chrome related to `requesAnimationFrame` which is supposed to run only when browser window is in focus. I'll discuss this issue with someone from Chrome team if they can suggest a workaround. But till then you can simply use `setTimeout` instead of `requestAnimationFrame` like so:

```
requestAnimationFrame(function () {
// some code
});
```

to

```
setTimeout(function () {
// some code here
}, 0);
```

**Comment by chinchang** [comment]
I fixed this in latest release: 0.2.0 :)

**Comment by itwars** [comment]
Excellent!
Many Thanks.
